tuning/data/data_config.py

```diff
@@ -153,6 +154,18 @@ def _validate_dataprocessor_config(dataprocessor_config) -> DataPreProcessorConf
         chat_template = kwargs["chat_template"]
         assert isinstance(chat_template, str), "chat_template should be a string"
         c.chat_template = chat_template
+    elif "chat_template_base64" in kwargs:
+        chat_template_base64 = kwargs["chat_template_base64"]
+        assert isinstance(
+            chat_template_base64, str
+        ), "chat_template_base64 should be a string"
+        logger.warning(
```

Comment:
you should also look to print the chat tempalte along side the warning...what was decoded one so users know what was picked up.

Reply:
> ...what was decoded one so users know what was picked up.
ok but it's not a string then we can't decode it right?